Makefile

```diff
@@ -25,6 +25,11 @@ docker-qgis-start: docker-up ## Start the containerized qgis
 	xhost +
 	docker-compose exec qgis sh -c 'DISPLAY=$$1 qgis' sh "unix$$DISPLAY"
 
+.PHONY: docker-qgis-test
+docker-qgis-test: docker-up ## Start the containerized qgis
+	xhost +
+	docker-compose exec qgis sh -c 'DISPLAY=$$1 ./scripts/qgis_testrunner.sh run_tests.run_test_modules' sh "unix$$DISPLAY"
```

Comment:
1. Use the scripts in /usr/bin, as per previous comments.
2. I think we should use xvfb instead of the actual user's display, for tests.

Reply:
now using `docker-compose exec qgis sh -c "/usr/bin/xvfb-run -- qgis_testrunner.sh tests_directory.run_tests.run_test_modules"
` for tests